content/en/blog/_posts/2024-08-13-Kubernetes-v1-31-Release.md

```diff
@@ -0,0 +1,429 @@
+becomes noticeable in clusters with tens of thousands of services).
+
+As of Kubernetes v1.31, the `nftables` mode is still relatively new, and may not be compatible with all network plugins; consult the documentation for your network plugin. 
+This proxy mode is only available on Linux nodes, and requires kernel 5.13 or later.
+Before migrating, note that some features, especially around NodePort services, are not implemented exactly the same in nftables mode as they are in iptables mode. 
+Check the [migration guide](/docs/reference/networking/virtual-ips/#migrating-from-iptables-mode-to-nftables) to see if you need to override the default configuration.
+
+This work was done as part of [KEP #3866](https://github.com/kubernetes/enhancements/issues/3866) by [SIG Network](https://github.com/kubernetes/community/tree/master/sig-network).
+
+### Changes to reclaim policy for PersistentVolumes
+
+The Always Honor PersistentVolume Reclaim Policy feature has advanced to beta in Kubernetes v1.31. 
+This enhancement ensures that the PersistentVolume (PV) reclaim policy is respected even after the associated PersistentVolumeClaim (PVC) is deleted, thereby preventing the leakage of volumes.
+
+Prior to this feature, the reclaim policy linked to a PV could be disregarded under specific conditions, depending on whether the PV or PVC was deleted first. 
+Consequently, the corresponding storage resource in the external infrastructure might not be removed, even if the reclaim policy was set to "Delete". 
+This led to potential inconsistencies and resource leaks.
+
+With the introduction of this feature, Kubernetes now guarantees that the "Delete" reclaim policy will be enforced, ensuring the deletion of the underlying storage object from the backend infrastructure, regardless of the deletion sequence of the PV and PVC.
+
+This work was done as a part of [KEP #2644](https://github.com/kubernetes/enhancements/issues/2644) and by [SIG Storage](https://github.com/kubernetes/community/tree/master/sig-storage).
+    
+### Bound service account token improvements
+
+The `ServiceAccountTokenNodeBinding` feature is getting promoted to beta in 1.31. 
+To allow for a robust chain of identity verification from the requester to the projected token, the Node object reference associated with the requesting Pod is now added in the private claims embedded into each JWT returned by the TokenRequest API. 
+This helps avoid replay attacks where forwarding of credentials of projected service account tokens from one actor to another. 
+We can cross-reference the identity of the caller to the Node reference embedded in the JWT, which allows this verification to be rooted upon the same root of trust as the kubelet/requesting entity. For more inforamtion, read the [bound service account tokens documentation](/docs/reference/access-authn-authz/service-accounts-admin/#bound-service-account-tokens).
```

Comment:
I would drop this line... it's not related to the ServiceAccountTokenNodeBinding feature that went beta in 1.31

Reply:
I've updated this